### PR TITLE
Update es.json

### PR DESCRIPTION
**Pull request by Samux0112**
I am samuel from ADOC

### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/ionic-bopis#contribution-guideline)

### CODE REVIEW

**Review by Samux0112 — COMMENTED** [review]
ok

File: src/locales/es.json [hunk 1]
```diff
@@ -1,307 +1,286 @@
...
+  "Field mapping name": "Nombre del mapeo de campos",
+  "File downloaded successfully": "Archivo descargado con éxito",
+  "File uploaded successfully": "Archivo cargado con éxito",
+  "Full Address": "Dirección Completa",
+  "Fulfillment capacity": "Capacidad de cumplimiento",
+  "Generate packing slip": "Generar documento de embalaje",
+  "Generate shipping label": "Generar etiqueta de envío",
+  "Go to OMS": "Ir a OMS",
+  "Go to Launchpad": "Ir a Launchpad",
+  "is identified as unfulfillable. other containing this product will be unassigned from this store and sent to be rebrokered.": "{ productName } se identifica como no cumplible.  { espacio } { orders } otros { orderText } que contienen este producto se desasignarán de esta tienda y se enviarán para ser revendidos.",
```
Comment (ymaheshwari1) [inline src/locales/es.json:100]: Please revert the translation for variables. All those entries that are in `{}`

File: src/locales/es.json [hunk 2]
```diff
@@ -1,307 +1,286 @@
...
-  "Turn off fulfillment for ": "Desactivar cumplimiento para {facilityName}",
-  "Unlimited": "Unlimited",
-  "Unlimited Capacity": "Unlimited Capacity",
-  "Unlimited fulfillment capacity": "Unlimited fulfillment capacity",
-  "Unlimited capacity removes the fulfillment capacity limit entirely. To add a fulfillment capacity to this facility, use the custom option.": "Unlimited capacity removes the fulfillment capacity limit entirely. To add a fulfillment capacity to this facility, use the custom option.",
-  "Unable to open as browser is blocking pop-ups.": "Unable to open { documentName } as browser is blocking pop-ups.",
-  "Unpack": "Desempacar",
-  "Unpacking this order will send it back to 'In progress' and it will have to be repacked.": "Desempacar este pedido lo enviará de vuelta a 'En curso' y tendrá que ser vuelto a empacar.",
-  "Update": "Update",
-  "Update mapping": "Update mapping",
```
Comment (ymaheshwari1) [inline src/locales/es.json:292]: Please provide the translations for labels those are missing.

File: src/locales/es.json [hunk 3]
```diff
@@ -1,307 +1,286 @@
...
-  ", and other products are identified as unfulfillable. These order items will be unassigned from this store and sent to be rebrokered.": "{ productName }, and { products } other products are identified as unfulfillable. { space } These order items will be unassigned from this store and sent to be rebrokered.",
-  ", and other products are identified as unfulfillable. other containing these products will be unassigned from this store and sent to be rebrokered.": "{ productName }, and { products } other products are identified as unfulfillable. { space } { orders } other { orderText } containing these products will be unassigned from this store and sent to be rebrokered.",
-  ", and other products were identified as unfulfillable. These items will be unassigned from this store and sent to be rebrokered.": "{ productName }, and { products } other products were identified as unfulfillable. { space } These items will be unassigned from this store and sent to be rebrokered."
+  "Store Id": "ID de la Tienda",
+  "Store Name": "Nombre de la Tienda",
+  "Store SKU": "SKU de la Tienda",
+  "Supplementary documents": "Documentos adicionales",
+  "Tax Exempt": "Exento de impuestos",
```
Comment (ymaheshwari1) [inline src/locales/es.json:250]: Please remove the extra entries from the file.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]
Hi @Samux0112 
Thanks for providing the translations.

Please check my comments once.

**Comment by ymaheshwari1** [comment]
Closing as handled in https://github.com/hotwax/fulfillment-pwa/pull/333

**Pull request closed without merging**